chef_master/source/aws_opsworks_chef_automate.rst

```diff
@@ -68,3 +68,96 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
 #. `Configure the audit cookbook <https://github.com/chef-cookbooks/audit#reporting-to-chef-visibility-via-chef-server>`_ to scan your nodes.
 
 .. note:: These instructions only detail what has to be added to the existing configuration as found in AWS OpsWorks for Chef Automate. For general instructions, see :doc:`integrate_compliance_chef_automate` (collector ``chef-server-visibility``) and :doc:`install_chef_automate`.
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate
+============================================================================
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate requires changes to the Chef Automate instance as well as the node that will run as a builder.
+
+#. Install Push Jobs Server on the Chef Automate instance by downloading the Push Jobs Server from https://downloads.chef.io/push-jobs-server and running:
```

Comment:
@srenatus - I COMPLETELY misread your message - I confused it for another issue.

So yes, I agree we should use install.sh. However, I get this error:

```
$ curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -c stable -P opscode-push-jobs-server
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 20500  100 20500    0     0   155k      0 --:--:-- --:--:-- --:--:--  156k
el 6 x86_64
Getting information for opscode-push-jobs-server stable  for el...
downloading https://omnitruck.chef.io/stable/opscode-push-jobs-server/metadata?v=&p=el&pv=6&m=x86_64
  to file /tmp/install.sh.11967/metadata.txt
trying wget...
ERROR 404
Omnitruck artifact does not exist for version  on platform el

Either this means:
   - We do not support el
   - We do not have an artifact for
```

Any thoughts on that?